src/seq/seqrecord.jl

```diff
@@ -31,6 +36,11 @@ function Base.getindex(seqrec::SeqRecord, r::UnitRange)
 end
 
 
+function Base.copy{T <: SeqRecord}(seqrec::T)
```

Comment:
I actually think that both things are annoying. Having to type `Module.` on every definition and importing things on other files. Importing the whole base (`importall Base`) is a bad decision?